ipynb/FilterAndPickOTUs.ipynb

```diff
@@ -0,0 +1,329 @@
+     "collapsed": false,
+     "input": [
+      "%run cluster_utils.ipy\n",
+      "import os\n",
+      "from functools import partial\n",
+      "\n",
+      "current_home = os.environ['HOME']\n",
+      "basedir = os.path.join(current_home, \"ICU\")\n",
+      "working_dir = os.path.join(basedir, 'blooming')\n",
+      "prj_name = 'blooming'\n",
+      "trim_length = 100\n",
+      "\n",
+      "# define a submit function against qsub\n",
+      "def submit(cmd_prefix, prj_name, queue, extra_args, cmd):\n",
+      "    \"\"\"Submission wrapper to submit to qsub\"\"\"\n",
+      "    cmd = \"%s; %s\" % (cmd_prefix, cmd)\n",
+      "    return submit_qsub(cmd, job_name=prj_name, queue=queue,\n",
+      "                       extra_args=extra_args)\n",
+      "\n",
+      "# define any environment specific details for any running compute job\n",
+      "cmd_prefix = 'source ~/.bash_profile; workon ag'\n",
+      "\n",
+      "# bootstrap the submit method\n",
+      "submit_8gb = partial(submit, cmd_prefix, prj_name, 'short8gb', '-l walltime=23:59:59')\n",
+      "submit_smr = partial(submit, cmd_prefix, prj_name, 'long8gb', '-l nodes=1:ppn=32 -l walltime=72:00:00')\n",
+      "\n",
+      "# setup a working directory\n",
+      "if not os.path.exists(working_dir):\n",
+      "    os.mkdir(working_dir)\n",
+      "path_joiner = lambda x: os.path.join(working_dir, x)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Here is where we'll setup paths for our input datasets"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "# setup the paths to your input datasets\n",
+      "paths_to_input_datasets = [\"/Users/mcdonadt/scratch/ICU/r1/seqs_r1.fna\",\n",
+      "                           \"/Users/mcdonadt/scratch/ICU/r2/seqs_r2.fna\"]\n",
+      "paths_to_input_metadata = [\"/Users/mcdonadt/scratch/ICU/minor_mapping.txt\",\n",
+      "                           \"/Users/mcdonadt/scratch/ICU/minor_mapping.txt\"]\n",
+      "fecal_category_and_value = [('BODY_SITE', 'UBERON:feces'),\n",
+      "                            ('BODY_SITE', 'UBERON:feces')]"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now, let's setup paths to our references"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "# path to the bloom sequences (note, this can be obtained from https://github.com/biocore/American-Gut/tree/master/data/AG\n",
+      "bloom_seqs = \"/Users/mcdonadt/ag/American-Gut/data/AG/BLOOM.fasta\"\n",
+      "\n",
+      "# path to Greengenes\n",
+      "reference_rep_set = '/scratch/Users/mcdonadt/greengenes/gg_13_8_otus/rep_set/97_otus.fasta' # e.g., path to 97_otus.fasta from Greengenes\n",
```

Comment:
This is how it was done in qiime 1.8 and it is more general specifying the files individually, unless I'm misunderstanding something?